integration_test/ops_agent_test.go

```diff
@@ -4542,6 +4543,64 @@ func TestNoNvmlOtelReceiverWithoutGpu(t *testing.T) {
 	})
 }
 
+func TestPartialSuccess(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		if gce.IsWindows(platform) {
+			t.SkipNow()
+		}
+		ctx, logger, vm := setupMainLogAndVM(t, platform)
+		logPath := logPathForPlatform(vm.Platform)
+
+		config := fmt.Sprintf(`logging:
+  receivers:
+    files_1:
+      type: files
+      include_paths: [%s]
+  service:
+    pipelines:
+      p1:
+        receivers: [files_1]`, logPath)
+
+		testFile, err := os.ReadFile(path.Join("partial_success", "test.log"))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if err = gce.UploadContent(ctx, logger, vm, bytes.NewReader(testFile), logPath); err != nil {
+			t.Fatal(err)
+		}
+		if err = agents.SetupOpsAgent(ctx, logger, vm, config); err != nil {
+			t.Fatal(err)
+		}
+
+		// scrape prometheus metrics from fluent-bit to see how many logs failed to send
```

Comment:
We shouldn't be testing this here IMO -- this should be tested upstream. In this test it's sufficient to use the same "log is missing logic" from e.g. `TestExcludeLogs` to verify that the giant logs are skipped while the small logs are present.